### DISCUSSION TITLE
Ne Kadar IBXC Coin alıcaz?

### DISCUSSION THREAD

**Discussion by huseyin03** [IBAX-io/go-ibax#529] [Rewards]
mesela ortalama 20-30bin puana ne kadar ibxc verilicek

**Comment by hes91** [comment]
300.000 IBXC ilk 1000 den sonrakilere eşit olarak paylaştırılacak.Yani net belli olan birşey yok.Katılımcı sayısına göre değişir.

**Reply by huseyin03** [reply]
ilk 1000 neye göre belirleniyor çekilişlemi yoksa referans puanları en çok olanlarmı

**Reply by CoinBaseCyprus** [reply]
Puana göre
